src/website/stores/Timestamp/TimestampActions.ts

```diff
@@ -0,0 +1,5 @@
+export default class TimestampActions {
```

Comment:
So for example, this would be `DataFilterActions`

Reply:
done